CONTRIBUTING.md

```diff
@@ -60,7 +60,7 @@ $ workon spyder-dev
 After you have created your development environment, you need to install Spyder's necessary dependencies. The easiest way to do so (with Anaconda) is
 
 ```bash
-$ conda install -c spyder-ide/label/dev --file requirements/conda.txt
+$ conda install -c spyder-ide/label/dev --file requirements/conda.txt python.app
```

Comment:
Please remove `python.app` from this line because is not available in other OSes and add below

    If you are on Mac, you also need to run

    ```bash
    $ conda install python.app
    ```